msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -549,7 +549,12 @@ await turnContext.SendActivityAsync(JsonConvert.SerializeObject(result));
 
```

Comment:
![image](https://user-images.githubusercontent.com/95664393/225490948-1c4c1498-7e7a-4af3-ae06-202a28384f56.png)

This should be replaced with something like "The bot will receive meeting start or end events automatically from all the meetings created in all the channels where the bot is a member, if ChannelMeeting.ReadBasic.Group is added to the manifest." permission."

Reply:
Hi @rm3f,

I've "If the `ChannelMeeting.ReadBasic.Group` permission is added to the manifest, the bot receives the meeting start or end events automatically from all the channel meetings created where the bot is added."

**We've removed the word member and replaced it with Added to a channel as when a bot is added to a channel, it's added as an app and also it doesn't show up in the list of members in a channel.**

Thanks,
Yash